scripts/lay-down-os

```diff
@@ -143,11 +137,8 @@ if [ -n ${ENV} ]; then
             ;;
         "rancher-upgrade")
             mount_device 
-            if [ -e ${BASE_DIR}/boot/grub/grub.cfg ]; then
-                grub2_config
-            else
-                pvgrub_config
-            fi
+            grub2_config
+            pvgrub_config
```

Comment:
Because what if the users stupidly deleted the grub config.  You wouldn't know which one to rebuild on upgrade.  Might as well do both.